kobo/client/__init__.py

```diff
@@ -289,58 +288,65 @@ def _login_krbv(self):
 
     def _login_gssapi(self):
         """Login using kerberos credentials (uses gssapi)."""
+
+        login_url = urlparse.urljoin(self._hub_url, "auth/krb5login/")
+
         # read default values from settings
         principal = self._conf.get("KRB_PRINCIPAL")
         keytab = self._conf.get("KRB_KEYTAB")
         service = self._conf.get("KRB_SERVICE")
         realm = self._conf.get("KRB_REALM")
         ccache = self._conf.get("KRB_CCACHE")
-        proxyuser = self._conf.get("KRB_PROXYUSER")
 
-        import enum
```

Comment:
Shouldn't gssapi import this if gssapi needs it?  I removed it since I couldn't identify any references to it in the code here.

Does "import enum" have the side-effect of making gssapi work?

Reply:
gssapi does import enum, so I don't get how this could be needed.